### PR TITLE
feat: Add security file 

### PR DESCRIPTION
**Pull request by choeqq**
As I saw - in each Vercel's repository there is `SECURITY.md` file, and since it was missing in this repo I've added it ✅ 

No Issues were assigned 😉

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+If you believe you have found a security vulnerability in SWR, we encourage you to let us know right away.
```
Comment (MaxLeiter) [inline .github/SECURITY.md:3]:
```suggestion
If you believe you have found a security vulnerability in the AI SDK, we encourage you to let us know right away.
```

Comment (choeqq) [reply .github/SECURITY.md]: my bad, fixed 😄

Comment (choeqq) [reply .github/SECURITY.md]: I've also added code of conduct file if you'll have a minute or so 😄 

https://github.com/vercel/ai/pull/672

**Comment by choeqq** [comment]
@MaxLeiter could you please take a look? 👀

**Review by MaxLeiter — APPROVED** [review]

**Pull request merged**